core/src/main/java/fj/test/Arbitrary.java

```diff
@@ -733,6 +733,10 @@ public StringBuilder f(final String s) {
     return arbitrary(listOf(aa.gen));
   }
 
+  public static <A> Arbitrary<NonEmptyList<A>> arbNonEmptyList(final Arbitrary<A> aa) {
+    return arbitrary(Gen.listOf1(aa.gen).map(list -> NonEmptyList.fromList(list).some()));
+  }
+
```

Comment:
No.
* Gen.listOf - list of arbitrary length (`[0, ∞)`).
* Gen.listOf1 - non empty list of arbitrary length (`[1, ∞)`).
* Gen.sequenceN - list of specified length.